shared/lib_utility_rate_equations.cpp

```diff
@@ -0,0 +1,901 @@
+#include "lib_utility_rate_equations.h"
```

Comment:
After the merge, should we refactor cmod_utilityrate5(or7) to use the libraries?